### ISSUE TITLE
Implement Datadog dashboard for MHV landing page and portal nav

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#82690]
### User story

As a Cartography team member, I want to have a Datadog dashboard that monitors a small number of key results/KPIs for products our team owns, so that we can monitor risks and begin understanding usage and usability of the integrated portal.

### Notes
- Dashboard defined in Analytics strategy ticket #82651
  - see [Mural area](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1716414036737/5aa77bac68425356bf314724636ed828f6bb5743?wid=0-1716421371873)
- If possible, consider implementing a custom Metric to preserve a KPI beyond the 30-day data retention window

Possible tasks:
- 

## Acceptance criteria

- Dashboard built in Datadog to monitor highest-priority 2-5 KPIs agreed upon with Robyn and Kaitlin
- PM review

**Comment by katbrinkley** [comment]
UX: Kat will meet with a developer to create datadog dashboard based on KPIs from monitoring plan

**Comment by katbrinkley** [comment]
Update: Needed a few more questions answered, Kat is to ask about 1 remaining widget. This will likely carry into the next sprint unless I can resolve tomorrow!

**Comment by wesrowe** [comment]
Updates from Fri 6/7:
I'm trying to get the "return to MHV Classic" clicks in Datadog. I can't find the action name anywhere. Argh.

1. Make sure there's data in DD – I went into Staging and clicked on a "Go back to the previous version of My HealtheVet" link several times. I think I clicked the one in the MR box, but it shouldn't matter.

2. Look for it in the Funnels UI – Soon after (15 mins?), I went into the Funnels UI looking for this click. It's not present in the action selection dropdown:

<details><summary>Screenshot</summary>

<img width="771" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/858fdb64-16f7-4ec0-a197-9ff0ceea45ba">

</details>

3. Look for it in the browser network inspector – I found a DD call with 5 json objects in it. The last one looked like the RUM object, but its action count was zero:

<details><summary>JSON</summary>

{
  "_dd": {
    "format_version": 2,
    "drift": 0,
    "session": {
      "plan": 1
    },
    "configuration": {
      "session_sample_rate": 100,
      "session_replay_sample_rate": 10
    },
    "document_version": 7
  },
  "application": {
    "id": "1f81f762-c3fc-48c1-89d5-09d9236e340d"
  },
  "date": 1717804205462,
  "service": "mhv-on-va.gov",
  "source": "browser",
  "session": {
    "id": "36838251-fb50-4446-a870-13e451ba6322",
    "type": "user"
  },
  "view": {
    "id": "07e9e488-b8ac-4acd-aa4e-4881ad662706",
    "url": "https://staging.va.gov/my-health/",
    "referrer": "https://staging.va.gov/my-va/?postLogin=true",
    "action": {
      "count": 0
    },
    "frustration": {
      "count": 0
    },
    "cumulative_layout_shift": 1.1749,
    "first_byte": 355200000,
    "dom_complete": 1133100000,
    "dom_content_loaded": 772800000,
    "dom_interactive": 560000000,
    "error": {
      "count": 0
    },
    "first_contentful_paint": 663200000,
    "first_input_delay": 2500000,
    "first_input_time": 17645000000,
    "is_active": false,
    "largest_contentful_paint": 1418200000,
    "load_event": 1137200000,
    "loading_time": 1137200000,
    "loading_type": "initial_load",
    "long_task": {
      "count": 3
    },
    "resource": {
      "count": 57
    },
    "time_spent": 80142000000,
    "in_foreground_periods": [
      {
        "start": 792900000,
        "duration": 17164900000
      },
      {
        "start": 25331600000,
        "duration": 142900000
      },
      {
        "start": 25477100000,
        "duration": 1587400000
      },
      {
        "start": 56932700000,
        "duration": 3099700000
      },
      {
        "start": 69589500000,
        "duration": 8415800000
      },
      {
        "start": 80065700000,
        "duration": 76300000
      }
    ]
  },
  "display": {
    "viewport": {
      "width": 851,
      "height": 919
    },
    "scroll": {
      "max_depth": 919,
      "max_depth_scroll_top": 0,
      "max_scroll_height": 3554,
      "max_scroll_height_time": 1820000000
    }
  },
  "type": "view",
  "privacy": {
    "replay_level": "mask-user-input"
  }
}

</details>

**Comment by katbrinkley** [comment]
From Anne CK:

[SM](https://app.ddog-gov.com/sb/f327ad72-c02a-11ec-a50a-da7ad0900007-0854b01ebd7d86bc121739b4f50b7450) , [MR](https://app.ddog-gov.com/sb/f327ad72-c02a-11ec-a50a-da7ad0900007-534a71ec89b6305ad34d5cdbe68b2ab7), and [Rx](https://app.ddog-gov.com/sb/f327ad72-c02a-11ec-a50a-da7ad0900007-d82308d1c6ca234057cfdedb5d0d2b38)

"SM is the only product in phase 1, so that dashboard is the most complete at this point. Rx will be getting some finessing soon and MR will be getting a makeover in August. MR is so big I may split it into domain-specific dashboards."

**Comment by wesrowe** [comment]
Wait a sec, @katbrinkley, we're actively working on this. Did you close it on accident?

**Comment by wesrowe** [comment]
big [slack thread](https://dsva.slack.com/archives/C0581MN69TJ/p1718035203474299) ongoing wherein we are trying to figure out how to get "Go back to previous version..." link clicks to show up in Datadog.

**Comment by wesrowe** [comment]
DD fixed itself in Prod for the "go back" clicks. The final piece of this is refining the "conversions" filter to exclude clicks on header components.

@katbrinkley mentioned in planning she has access and could take a pass at this. Lmk if you were successful or need support, Kat!

**Comment by katbrinkley** [comment]
I might need some brainstorming on this with someone.. we have this widget labeled as "Clicks on main page area", but I'm not sure if this should be a "conversion" since they aren't completing a goal. (Note: The conversion rate is the number of visitors to your website that completed a desired goal (a conversion) out of the total number of visitors. - DD definition). If we're just measuring clicks, I think that engagement makes more sense (which we already have a widget for - "Clicks We Want" under MHV Home Details section.

@wesrowe does this make sense to you?

**Comment by wesrowe** [comment]
@katbrinkley, the "clicks we want" widget is intended to display what specific clicks were counted as conversions. The main % widget is the actual KPI. Both are related, but different. Also, we discussed enough at the time and settled on this measure for "conversions", since our page's goal (it does have one) is to get users to where they need to go. I still think it's the best measure of conversions, and IMHO much more useful than the Engagement number we have. (But that one is standard across apps so is useful for comparison.)

**Comment by katbrinkley** [comment]
Updates:
- Met with Florence about verifying all the links we wanted to include/exclude on the Home Page Clicks We Want widget. We found that there were many links we were missing to include, some we had questions about, some we needed to remove.
- I created a [spreadsheet](https://docs.google.com/spreadsheets/d/1U3W3aZ6hLhVzm1A2bVV-ibjb0bHzK_4oUTX2Iq9815A/edit?usp=sharing) of all the links in the main page that we want to track, including the ones Florence caught that we needed to add.
- I created a new widget based on the clicks to *include rather than exclude. Placed it in the boneyard for now.
- Will set up a meeting with the team to review this and make sure we have everything.
